### PR TITLE
Move MetricsManager from Indy

### PR DESCRIPTION
**Pull request by ruhan1**
Remove Indy specific code and come up with DefaultMetricsManager.

### CODE REVIEW

File: metrics/core/src/main/java/org/commonjava/o11yphant/metrics/DefaultMetricsManager.java [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    private MetricsConfig config;
+
+    private Random random = new Random();
+
+    @PostConstruct
+    public void init()
+    {
+        if ( !config.isEnabled() )
+        {
+            logger.info( "Indy metrics subsystem not enabled" );
```
Comment (ligangty) [inline metrics/core/src/main/java/org/commonjava/o11yphant/metrics/DefaultMetricsManager.java:91]: Remove "indy" from log info?

File: metrics/core/src/main/java/org/commonjava/o11yphant/metrics/DefaultMetricsManager.java [hunk 2]
```diff
@@ -0,0 +1,296 @@
+/**
+ * Copyright (C) 2011-2020 Red Hat, Inc. (https://github.com/Commonjava/indy)
```
Comment (ligangty) [inline metrics/core/src/main/java/org/commonjava/o11yphant/metrics/DefaultMetricsManager.java:2]: Fix git url?

File: metrics/core/src/main/java/org/commonjava/o11yphant/metrics/DefaultMetricsManager.java [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+    private final Logger logger = LoggerFactory.getLogger( getClass() );
+
+    @Inject
+    private MetricRegistry metricRegistry;
+
+    @Inject
+    private HealthCheckRegistry healthCheckRegistry;
+
+    @Inject
+    private Instance<AbstractHealthCheck> indyHealthChecks;
```
Comment (ligangty) [inline metrics/core/src/main/java/org/commonjava/o11yphant/metrics/DefaultMetricsManager.java:73]: Var name?

File: metrics/core/src/main/java/org/commonjava/o11yphant/metrics/DefaultMetricsManager.java [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+
+    @Inject
+    private Instance<CompoundHealthCheck> indyCompoundHealthChecks;
```
Comment (ligangty) [inline metrics/core/src/main/java/org/commonjava/o11yphant/metrics/DefaultMetricsManager.java:76]: Var name?

Comment (ruhan1) [reply metrics/core/src/main/java/org/commonjava/o11yphant/metrics/DefaultMetricsManager.java:76]: Cool. those slipped over my eyes. Will fix them all.

**Comment by ruhan1** [comment]
@ligangty I will fix the word "indy" in var/license in next pr.

**Pull request merged**